internal/removals/remove.go

```diff
@@ -132,8 +137,12 @@ func GetRemainingMap(logger logr.Logger, adminClient fdbadminclient.AdminClient,
 			continue
 		}
 
-		for _, pAddr := range processGroup.Addresses {
-			addresses = append(addresses, fdbv1beta2.ProcessAddress{IPAddress: net.ParseIP(pAddr)})
+		if fdbVersion.IsAtLeast(fdbv1beta2.Versions.LatestFdbVersion) {
+			addresses = append(addresses, fdbv1beta2.ProcessAddress{StringAddress: processGroup.GetExclusionString()})
```

Comment:
Should we also add the IP addresses here? See my comment above for the background.